javaparser-core/src/main/java/com/github/javaparser/ParserConfiguration.java

```diff
@@ -218,7 +218,7 @@ public boolean isYieldSupported() {
     private boolean preprocessUnicodeEscapes = false;
     private SymbolResolver symbolResolver = null;
     private int tabSize = 1;
-    private LanguageLevel languageLevel = JAVA_8;
+    private LanguageLevel languageLevel = POPULAR;
```

Comment:
I'm not sure why I ever set this to Java 8, it seems more logical to have the most popular Java version as the default.